app/main.py

```diff
@@ -1,3 +1,66 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float | int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        if isinstance(self.km, float):
+            return f"Distance: {self.km} kilometers."
+        elif isinstance(self.km, int):
+            return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        if isinstance(self.km, float):
+            return f"Distance(km={self.km})"
+        elif isinstance(self.km, int):
+            return f"Distance(km={self.km})"
```

Comment:
Similar to the `__str__` method, the `else` statement here is unnecessary. You can directly return the string without using `elif` since the previous condition already returns a value if true.